recipes/leftbar/style/leftbar_row.less

```diff
@@ -82,6 +82,40 @@
     --leftbar-row-omega-min-width: var(--leftbar-row-alpha-width);
   }
 
+  &--typing {
+    height: 24px;
+    width: 24px;
+    position: absolute;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 2px;
+    border-radius: var(--br-pill);
+    opacity: 0.75;
+  }
+
+  &--typing span {
+    transition: all 500ms ease;
+    background: var(--white);
```

Comment:
Color combination TBD via @bhawkins-dialpad per https://dialpad.slack.com/archives/C04QQ1GKT96/p1678310674717059. Just ensure design token usage to future-proof dark theme.